### PR TITLE
Add an External Display application for viewing gff/gtf files at IGV.

### PR DESCRIPTION
**Pull request by blankenberg**

### CODE REVIEW

**Comment by blankenberg** [comment]
ping @nekrut

File: display_applications/igv/gff.xml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    &lt;resources os=&quot;Linux&quot;&gt;
+        &lt;nativelib href=&quot;hdfnative-linux64.jar&quot;/&gt; 
+    &lt;/resources&gt;
+
+  &lt;application-desc main-class=&quot;org.broad.igv.ui.IGVMainFrame&quot;&gt;
+     &lt;argument&gt;-g&lt;/argument&gt;
+     &lt;argument&gt;${site_organism}&lt;/argument&gt;
+     &lt;argument&gt;${gff_file.url}&lt;/argument&gt;
+  &lt;/application-desc&gt;
+&lt;/jnlp&gt;
```
Comment (hexylena) [inline display_applications/igv/gff.xml:83]: Is CDATA an option here? Might make this more legible?

Comment (blankenberg) [inline display_applications/igv/gff.xml:83]: That is probably an option, but then I'd have to do more than copy&paste with some modifications from the other known-working IGV displays.

Comment (hexylena) [inline display_applications/igv/gff.xml:83]: hah, fair enough.

**Pull request merged**